### PR TITLE
chore: Adds a some profile fields, including avatar and addresses

### PR DESCRIPTION
**Pull request by kiwicopple**

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -35,7 +37,11 @@ create table customers (
   -- UUID from auth.users
   id uuid references auth.users not null primary key,
   -- The user's customer ID in Stripe. User must not be able to update this.
-  stripe_customer_id text
+  stripe_customer_id text,
+  -- The customer's billing address, stored in JSON format
+  billing_address jsonb,
+  -- Stores your customer's payment instruments.
+  payment_method jsonb
```
Comment (kiwicopple) [inline schema.sql:44]: https://stripe.com/docs/api/payment_methods

Should this be a list? i.e. `payment_methods`. Looks like there can be several methods/cards

File: schema.sql [hunk 2]
```diff
@@ -6,7 +6,9 @@ create table users (
   -- UUID from auth.users
   id uuid references auth.users not null primary key,
   first_name text,
-  last_name text
+  last_name text,
+  -- The URL for the user's avatar. Images should be stored outside of the database.
+  avatar_url text default "/avatar.png",
```
Comment (kiwicopple) [inline schema.sql:11]: As discussed, can be done with blobs or [hashes](https://github.com/woltapp/blurhash), but simple is good

Comment (leerob) [reply schema.sql:11]: If we want to simplify this, we can use https://unavatar.now.sh/

Comment (kiwicopple) [reply schema.sql:11]: that's very cool. @thorwebdev - will leave that decision to you - we can remove the field if we use unavatar (will everyone have email though?). 

Default to https://unavatar.now.sh/facebook/zuck so they are incentivised to update their profile

Comment (thorwebdev) [reply schema.sql:11]: Happy to use https://unavatar.now.sh/. @leerob would we put that straight into the source code? Then we can remove the avatar_url column here, correct?

File: schema.sql (hunk 1 above)
Comment (thorwebdev) [reply schema.sql:44]: there's only one default_payment method, that's what we'll use here 👍

File: schema.sql (hunk 2 above)
Comment (leerob) [reply schema.sql:11]: If we can't figure out their avatar based on their email, then we should use the gradient circle like here -> https://demo.vercel.store/

Comment (thorwebdev) [reply schema.sql]: @leerob okay, I've removed it from the schema. Can I leave the avatar to you?

File: components/Navbar/Navbar.js [hunk 3]
```diff
@@ -17,13 +20,24 @@ const Navbar = () => {
...
             </nav>
           </div>
 
           <div className="flex flex-1 justify-end space-x-8">
-            <Link href="/account">
-              <a className={s.link}>Account</a>
-            </Link>
+            {user ? (
+              <Link href="#">
+                <a className={s.link} onClick={() => supabase.auth.signOut()}>
```
Comment (leerob) [inline components/Navbar/Navbar.js:32]: I'd almost think this should be part of the useAuth hook. Maybe later.

**Review by leerob — APPROVED** [review]

File: schema.sql (hunk 2 above)
Comment (leerob) [reply schema.sql]: Yep, I can do that. When's your deadline again?

Comment (thorwebdev) [reply schema.sql]: End of this week would be stellar. Then I can record the workshop over the weekend. Do you think it's feasible to have the project moved to the Vercel GitHub org by then?

Comment (leerob) [reply schema.sql]: Yeah I think so.

**Pull request merged**